admin.py

```diff
@@ -0,0 +1,183 @@
+def choose_the_option():
+    option = int(input("Вибери спеціальність: \n" "1. Адмін\n"))
+    return option
+
+
+option = choose_the_option()
+
+drinks = {
+    "Лагер": 30,
+    "Ель": 35,
+    "Оболонь": 28,
+    "Чернігівське": 30,
+    "Львівське": 32,
+    "Обрій": 31,
+    "Горілка-Хортиця": 180,
+    "Горілка-Хлібна слобода": 200,
+    "Горілка-Вознесенська": 150,
+}
+
+
+def function_of_admin():
```

Comment:
You are in the admin.py file, you don't need to specify that this is an admin function. Need to rename.

Reply:
I don't think it's necessary to rename the name of the admin function. The name of the function should describe its actions.